projects/Mallard/src/helpers/words.ts

```diff
@@ -52,6 +52,11 @@ export const BETA_PROGRAMME_FAQ_HEADER_TITLE = 'Beta Programme FAQ'
 export const REFRESH_BUTTON_TEXT = 'Refresh'
 export const DOWNLOAD_ISSUE_MESSAGE_OFFLINE = `You're currently offline. You can download it when you go online`
 export const CUSTOMER_HELP_EMAIL = 'customer.help@theguardian.com'
+export const CREDITS_HEADER_TITLE = 'Credits'
```

Comment:
Although the titles seems shorter than the const names, there is inconsistency of usage throughout the screens. This keeps it in line.